### ISSUE TITLE
package.json specifies a `main` module field that could not be resolved

### ISSUE THREAD

**Issue by macfanatic** [scriptex/typed-usa-states#46]
Looks like the published npm package references a "dist" folder that's not part of the included files.

Using [v1.1.0](https://www.npmjs.com/package/typed-usa-states/v/1.1.0) with yarn.

```
While trying to resolve module `typed-usa-states` from file `/Users/matt/Projects/perfect-measure-mobile-app/src/db/models/address.ts`, the package `/Users/matt/Projects/perfect-measure-mobile-app/node_modules/typed-usa-states/package.json` was successfully found. However, this package itself specifies a `main` module field that could not be resolved (`/Users/matt/Projects/perfect-measure-mobile-app/node_modules/typed-usa-states/dist/index.js`. Indeed, none of these files exist:

  * /Users/matt/Projects/perfect-measure-mobile-app/node_modules/typed-usa-states/dist/index.js(.native|.ios.expo.ts|.native.expo.ts|.expo.ts|.ios.expo.tsx|.native.expo.tsx|.expo.tsx|.ios.expo.js|.native.expo.js|.expo.js|.ios.expo.jsx|.native.expo.jsx|.expo.jsx|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx|.ios.js|.native.js|.js|.ios.jsx|.native.jsx|.jsx|.ios.json|.native.json|.json|.ios.wasm|.native.wasm|.wasm)
  * /Users/matt/Projects/perfect-measure-mobile-app/node_modules/typed-usa-states/dist/index.js/index(.native|.ios.expo.ts|.native.expo.ts|.expo.ts|.ios.expo.tsx|.native.expo.tsx|.expo.tsx|.ios.expo.js|.native.expo.js|.expo.js|.ios.expo.jsx|.native.expo.jsx|.expo.jsx|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx|.ios.js|.native.js|.js|.ios.jsx|.native.jsx|.jsx|.ios.json|.native.json|.json|.ios.wasm|.native.wasm|.wasm)
Failed building JavaScript bundle.
```

**Comment by scriptex** [comment]
Thanks for reporting this. Sounds like a really weird issue. I will investigate when I have some free time.

**Comment by scriptex** [comment]
@macfanatic I have released a new version (v1.1.1) which fixes this issue. Please use this version by installing it via npm or yarn. Once again, thanks for reporting this!
